components/LoaderSpinner/LoaderLine.js

```diff
@@ -0,0 +1,19 @@
+import React, { useEffect } from "react";
+import { LoadingLineHigh, LoadingLineLow } from "./LoaderLine.styled";
+
+export default function LoaderLine() {
+  useEffect(() => {
+    const timer = setTimeout(() => {
+      window.location.reload(); // refresh the page
+    }, 5000); // after 5 sec.
+
+    return () => clearTimeout(timer);
+  }, []);
```

Comment:
Forgot to remove this. This would be part of the Loader component that I was trying to implement but that I didn't leave in the code. The intention was also to give a false period of time in which the user visualizes something without realizing that the response to the API takes so long. I have to look for a better solution for this.